src/js/contentscripts/socialwidgets.js

```diff
@@ -288,6 +279,64 @@ function replaceSubsequentTrackerButtonsHelper(trackerDomain) {
   });
 }
 
+function createReplacementWidget(button, buttonToReplace, trackerUrls) {
+  let widgetFrame = document.createElement('iframe');
+
+  // widget replacement frame styles
+  let styleAttrs = [
+    "border: 1px solid #ec9329",
+    "padding: 10px",
```

Comment:
I think this line is causing the overflow issues on https://quakesepicenter.com/2018/03/22/three-tactical-observations-from-san-joses-3-2-defeat-to-sporting-kansas-city/ and https://oartistadodia.blogspot.com/2018/03/acerca-do-gabinete-de-crise.html. We can either remove the padding altogether or set the width/height to be `clientWidth/Height - 10`.